test/src/596-catalogbalancer/main

```diff
@@ -0,0 +1,156 @@
+cvmfs_test_name="Catalog balancer"
+cvmfs_test_autofs_on_startup=false
+
+add_files() {
+  local repo_dir=$1
```

Comment:
No, it's the absolute path in the system, not in the repository. That function simply adds files in the directory you specify, and is not related at all with any repository, and doesn't understand any logic about them